trello.py

```diff
@@ -0,0 +1,60 @@
+from config import KEY, TOKEN, BoardID, TTD, DO
+import requests 
+import json
+
+class Item:
+    def __init__(self, id, status, title):
+        self.id = id
+        self.title = title
+        self.status = status
+
+
+def fetch_all_items():
+    params = (
+        ('key', KEY),
+        ('token', TOKEN),
+        ('fields', 'name,idList')
+    )
+
+    r = requests.get('https://api.trello.com/1/boards/5eef26d3f3a754437122f88b/cards', params=params)
+    data = r.json()
+    item_list = []
+    for card in data:
+        if card['idList'] == TTD:
+            card['idList'] = 'Not Started'
+        elif card['idList'] == DO:
+            card['idList'] = 'Complete'
+
+        if [item for item in item_list if item.id == card['id']]:
+            pass
+        else:
+            item_list.append(Item(id=card['id'], status=card['idList'], title=card['name']))
```

Comment:
Might be worth extracting to a conversion `@classmethod` on the `Item` class (call `from_trello_card` or something similar)